crates/config/src/config.rs

```diff
@@ -381,6 +381,7 @@ timeout = -4096
 bad_protocol = -2147483648
 failed_to_connect = -25600
 dropped = -4096
+bad_announcement = -1204
```

Comment:
is this change safe? @mattsse

Reply:
ok good, was wondering since the test is called something 'check backwards compatible' haha